tests/test_score_justification_endpoints.py

```diff
@@ -62,31 +62,34 @@ def test_score_justification_is_created(self, flask_test_client):
         assert response.status_code == 201
         assert score_justification.get("score") == 1
 
-    def test_scores(self, flask_test_client):
-        expected_response = [
-            {
-                "criteria_id": str(SqliteTestDB.crit_1_uuid),
-                "criteria_name": "strategy",
-                "total_score": 5,
-                "weight": 0.3,
-                "weighted_score": 1.5,
-            },
-            {
-                "criteria_id": str(SqliteTestDB.crit_2_uuid),
-                "criteria_name": "deliverability",
-                "total_score": 6,
-                "weight": 0.4,
-                "weighted_score": 2.4000000000000004,
-            },
-            {
-                "criteria_id": str(SqliteTestDB.crit_3_uuid),
-                "criteria_name": "value_for_money",
-                "total_score": 5,
-                "weight": 0.3,
-                "weighted_score": 1.5,
-            },
-        ]
-        assessment_id = str(SqliteTestDB.assessment_2_uuid)
-        endpoint = f"/assessments/{assessment_id}/scores"
-        response = flask_test_client.get(endpoint)
-        assert response.json["scores"] == expected_response
+
+# TODO reinstate this when we go back to assessment. Currently points at data
```

Comment:
should we create a tech debt ticket for this?

Reply:
Good shout, added here: https://digital.dclg.gov.uk/jira/browse/FS-1344